### PR TITLE
Update change-account-information.md

### PR DESCRIPTION
**Pull request by ReneeRendon**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-cb4262f495/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-84eceecb10/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-31afa4e863/how-to/).

**Comment by ReneeRendon** [comment]
JIRA 328

**Comment by stephamon** [comment]
Hey @ReneeRendon, here are my edits. Let me know if you have any questions!

[Change.account.information_SFedits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1271384/Change.account.information_SFedits.docx)
[Change.account.information_SFedits.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1271385/Change.account.information_SFedits.pdf)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6af5d69c58/how-to/)

File: content/general/change-account-information.md [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
+You can download the **Service Transfer** form directly from
 the [Cloud Control Panel](https://mycloud.rackspace.com/) by using the following steps.
 
 **Note:** If you are a Dedicated Hosting customer, you can request this form
 directly from your Account Manager.
 
-1. In the upper-right corner of the Cloud Control Panel, click the ***userName*** menu and select **Documents and Forms**.
-2.  In the **Rackspace Account Change Forms** section, click the **Assignment and Assumption Form** link.
-3.  Download the **Assignment and Assumption** form and complete it.
+1.  In the upper-right corner of the Cloud Control Panel, click the your username, and then click **Documents and Forms**.
```
Comment (stephamon) [inline content/general/change-account-information.md]: "click your username" not "click **the** your username"

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-1efe059485/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**